cmd/loop/liquidity.go

```diff
@@ -0,0 +1,174 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/lightninglabs/loop/looprpc"
+	"github.com/urfave/cli"
+)
+
+var getLiquidityParamsCommand = cli.Command{
+	Name:  "getparams",
+	Usage: "show liquidity manager parameters",
+	Description: "Displays the current set of parameters that are set " +
+		"for the liquidity manager.",
+	Action: getParams,
+}
+
+func getParams(ctx *cli.Context) error {
+	client, cleanup, err := getClient(ctx)
+	if err != nil {
+		return err
+	}
+	defer cleanup()
+
+	cfg, err := client.GetLiquidityParams(
+		context.Background(), &looprpc.GetLiquidityParamsRequest{},
+	)
+	if err != nil {
+		return err
+	}
+
+	printRespJSON(cfg)
+
+	return nil
+}
+
+var setLiquidityParamCommand = cli.Command{
+	Name:        "setparam",
+	Usage:       "set liquidity manager rule for a channel",
+	Description: "Update or remove the liquidity rule for a channel.",
+	ArgsUsage:   "shortchanid",
+	Flags: []cli.Flag{
+		cli.IntFlag{
+			Name: "mininbound",
+			Usage: "the minimum percentage of inbound liquidity " +
+				"to total capacity beneath which to " +
+				"recommend loop out to acquire inbound.",
+		},
+		cli.IntFlag{
+			Name: "minoutbound",
```

Comment:
This, or is `min_balance` / `max_balance` more clear? Maybe it is also good to connect with how LiT communicates about channel state.

Reply:
I don't think "balance" is the best terminology to use here since that implies absolute values to me instead of percentages. Since it's a threshold rule, it could be called `inbound_threshold`/`outbound_threshold` for example?